pkg/kubelet/dockershim/docker_container.go

```diff
@@ -217,12 +221,26 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get container %q log path: %v", containerID, err)
 	}
-	if path != "" {
+
+	dockerLoggingDriver := ""
+	dockerInfo, err := ds.client.Info()
+	if err != nil {
+		glog.Errorf("failed to execute Info() call to the Docker client: %v", err)
+	} else {
+		dockerLoggingDriver = dockerInfo.LoggingDriver
+		glog.V(10).Infof("docker logging driver is %s", dockerLoggingDriver)
+	}
+
+	if path != "" && realPath != "" {
 		// Only create the symlink when container log path is specified.
 		if err = ds.os.Symlink(realPath, path); err != nil {
 			return fmt.Errorf("failed to create symbolic link %q to the container log file %q for container %q: %v",
 				path, realPath, containerID, err)
 		}
+	} else if dockerLoggingDriver == dockerDefaultLoggingDriver {
+		glog.Errorf("cannot create symbolic link because container log file doesn't exist!")
```

Comment:
s/cannot/Cannot

Usually, log output should start with capital letter, error starts with small letter.